esphome/components/inkbird_ibsth1_mini/sensor.py

```diff
@@ -0,0 +1,37 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, esp32_ble_tracker
+from esphome.const import CONF_BATTERY_LEVEL, CONF_HUMIDITY, CONF_MAC_ADDRESS, CONF_TEMPERATURE, \
+    UNIT_CELSIUS, ICON_THERMOMETER, UNIT_PERCENT, ICON_WATER_PERCENT, ICON_BATTERY, CONF_ID
+
+DEPENDENCIES = ['esp32_ble_tracker']
```

Comment:
Please add youself to the CODEOWNERS and then run `script/build_codeowners.py`

```suggestion
CODEOWNERS = ['@fkirill']
DEPENDENCIES = ['esp32_ble_tracker']
```

Reply:
Hey Jesse, added CODEOWNERS, regenerated the codeowners index.